sideboard/lib/__init__.py

```diff
@@ -16,9 +16,10 @@
            'ConfigurationError', 'parse_config',
            'is_listy', 'listify', 'serializer', 'cached_property', 'class_property', 'entry_point',
            'stopped', 'on_startup', 'on_shutdown', 'mainloop', 'ajax', 'renders_template', 'render_with_templates',
+           'restricted', 'all_restricted', 'register_authenticator'
            'DaemonTask', 'Caller', 'GenericCaller', 'TimeDelayQueue',
            'WebSocket', 'Model', 'Subscription', 'MultiSubscription',
-           'listify', 'serializer', 'cached_property', 'request_cached_property', 'is_listy', 'entry_point',
+           'listify', 'serializer', 'cached_property', 'request_cached_property', 'is_listy', 'entry_point', 'RWGuard',
```

Comment:
Oh huh, I think this is the merge problem.  We already merged the ``RWGuard`` into master I believe; it started as a branch off a branch and then got merged into the branch before being merged into master, so I think that's why we're having merge problems.  None of the ``RWGuard`` stuff is actually new and you can see the original review at #46